tasks/shell_crmsh/crm-cluster-properties.yml

```diff
@@ -7,9 +7,8 @@
   ansible.builtin.set_fact:
     __ha_cluster_property_attr: "{{
       (properties_set.attrs + [{'name': 'stonith-enabled', 'value': 'true'}])
-      if properties_set.attrs | selectattr('name','equalto','stonith-enabled')
-       | list | length == 0
-      else properties_set.attrs }}"
+      if properties_set.attrs | selectattr('name', 'equalto', 'stonith-enabled') | list | length == 0
```

Comment:
```suggestion
      if properties_set.attrs
      | selectattr('name', 'equalto', 'stonith-enabled') | list | length == 0
```

Reply:
This is problem with our 80 limit, making it difficult to read.

Reply:
It depends, for those who have codespace limited to 80 symbols and need to scroll horizontally to get the full line, reading long lines is a bigger problem. I prefer complying with this requirement where possible.

Reply:
completed in latest commit